index.html

```diff
@@ -2,15 +2,88 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <title>Karma</title>
     <link href='//fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
     <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
     <link rel="stylesheet" href="css/normalize.css">
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.9/css/all.css" integrity="sha384-5SOiIsAziJl6AWe0HWRKTXlfcSHKmYV4RBF18PPJ173Kzn7jzMyFuTtk8JA7QQG1" crossorigin="anonymous">
+    
 </head>
 <body>
 
+        <nav class="navigation">
+     <div class="content">
+        <img class="logo" src="img/karma-logo.svg"  width="26">
+       <ul class="links">
+         <li class="active">Meet Karma</li>
+         <li>How it Works</li>
+         <li>Store</li>
+         <li>Blog</li>
+         <li>Help</li>
+         <li>Login</li>
+       </ul>
+       
+     </div>
+   </nav>
+   <section class="intro">
+     <div class="intro">
```

Comment:
Watch out, you have 2 instances of the `.intro` class so the background image is appearing twice.